index.html

```diff
@@ -187,20 +187,89 @@ <h2>Name and Description</h2>
+    <dd>name is generated from the <a class="termref" data-lt="text node">Text nodes</a> associated with the <a class="termref">element</a>. Although this may be allowed in addition to "author" in some <a class="termref">roles</a>, "content" is used only if higher priority "author" features are not provided. Priority is defined by the <a href="#mapping_additional_nd_te">text content computation</a> algorithm.</dd>
   </dl>
   <p>The <cite><a class="specref" href="#">Accessible Rich Internet Applications (WAI-ARIA) 1.2</a></cite> [[!WAI-ARIA]] specification provides lists of roles that support <a class="specref" href="#namefromauthor">name from author</a> and <a class="specref" href="#namefromcontent">name from content</a>. </p>
   <section id="mapping_additional_nd_name">
     <h3>Name Computation</h3>
-    <p><a class="termref">User agents</a> MUST compute an <a class="termref">accessible name</a> using the rules outlined below in the section titled <a href="#mapping_additional_nd_te">Accessible Name and Description Computation</a>.</p>
+    <p><a class="termref">User agents</a> MUST compute an <a class="termref">accessible name</a> using the rules outlined below in the section titled <a href="#mapping_additional_nd_te">Text Content Computation</a>.</p>
   </section>
+
   <section id="mapping_additional_nd_description">
     <h3>Description Computation</h3>
-    <p>If <a class="property-reference" href="#aria-describedby"><code>aria-describedby</code></a> is present, <a class="termref">user agents</a> MUST compute the accessible description by concatenating the text alternatives for elements referenced by an <code>aria-describedby</code> attribute on the current element. The text alternatives for the referenced elements are computed using a number of methods, outlined below in the section titled <a href="#mapping_additional_nd_te">Accessible Name and Description Computation</a>. </p>
+    <p>The following table provides the order of precedence for markup that can be applied to compute an <a class="termref">accessible description</a>. <a class="termref">User agents</a> MUST use the first applicable entry from the table where the listed conditions are met, as described in the last column. The user agent MUST NOT use any markup other that the first relevant markup found, even if that markup results in an empty description:</p>
+    <table>
+      <thead>
+        <tr>
+          <th>Precedence</th>
+          <th>Attribute</th>
+          <th>Applicable conditions</th>
+          <th>How used to compute description</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <th>1</th>
+          <td><a class="property-reference" href="#aria-describedby"><code>aria-describedby</code></a> attribute</td>
+          <td>Use on any element</td>
+          <td><a href="#mapping_additional_nd_te">Text content computation</a> on any relevant nodes, concatenated, and separated by a space character</td>
```

Comment:
"any relevant nodes" seems a little vague.
```suggestion
          <td><a href="#mapping_additional_nd_te">Text content computation</a> on all nodes referenced by aria-describedby on the element, concatenated, and separated by a space character</td>
```

Reply:
That's fine. We may as well be as exact as possible where it's not too verbose. In general, I think we should work to make this spec more concise and readable for browser developers.